src/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -154,7 +154,11 @@ public static String getParentPath(String remotePath) {
     public static OCFile fillOCFile(RemoteFile remote) {
         OCFile file = new OCFile(remote.getRemotePath());
         file.setCreationTimestamp(remote.getCreationTimestamp());
-        file.setFileLength(remote.getLength());
+        if (remote.getMimeType().equalsIgnoreCase("DIR")){
+            file.setFileLength(remote.getSize());
```

Comment:
LOL 

Was checking the library, we explicitly request the ```oc:size``` property from https://github.com/owncloud/android-library/pull/55 . We should make some minimum test with server 8.0.

Also would be wise to check what minimum OC server we grant the app works with. Do you recall it, @rperezb ?